lib/users.js

```diff
@@ -38,35 +38,22 @@ users.sendPasswordResetEmail = async (userAccount) => {
 };
 
 users.signUp = async (userInfo) => {
-  try {
-    await schema.validate(userInfo);
-  } catch (err) {
-    throw new Error(err);
-  }
-
-  const { email } = userInfo;
-  const { Accounts } = db.getModels();
-  const userAccount = await Accounts.findOne({
-    where: {
-      email: email,
-    },
-  });
+  await schema.validate(userInfo);
   const username = uniqueNamesGenerator({
     length: 2,
-    dictionaries: [adjectives, colors, animals],
```

Comment:
I took `colors` out because it only uses the first 2 dictionaries since the length is 2